tracpro/settings/base.py

```diff
@@ -333,7 +333,7 @@ def _org_scheduler_task(task_name):
         'delete',  # can delete an object
         'list',  # can view a list of the objects
     ),
-    'orgs.org': ('create', 'update', 'list', 'edit', 'home'),
+    'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'fetch_runs'),
```

Comment:
@rowanseymour `python manage.py migrate` still has no feedback after adding this to the PERMISSIONS setting. And going to this page as an administrator still brings me to the login page. 

Am I missing something else? Does this permission work on SmartFormView?